### PR TITLE
Release 2.1.3

### PR DESCRIPTION
**Pull request by sjieg**
- NEW: We now support adding multiple configuration files by using `LapisLazuli::WorldModule::Config.add_config("config/config.yml")`
- DEPRICATED: `LapisLazuli::WorldModule::Config.config_file = "config/config.yml"`
- NEW: The generated template using `lapis_lazuli create <projectname>` has been completely reworked up to the new standards. See below for more details.
- NEW: Updated Lapis Lazuli dependencies, now allowing Cucumber 3 (Beta)
- NEW: Opened browser now defaults to whatever `Watir` open when no `:browser` option is given. (Currently Chrome, this was Firefox)
- NEW: Now depending on `deep_merge` to be able to smart-merge 2 config files, with the same values together.
- FIX: Resizing browser would sometime not work.
- FIX: `env_or_config()` did not throw an error if neither was found and no default was given.
- FIX: `get_http_status` properly returns the correct status code

UPDATED TEMPLATE:
- More detailed `cucumber.yml`
- Split up the feature files: 
  - Beginner: `basic.feature` (Includes a debug practise scenario)
  - Advanced: `account.feature`
  - Practise: `todo.feature`
- Added helper modules, modules made to simplify your tests.
- Added login / logout / register scenario's.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+image: spritecloud/ruby-selenium-runner:2.4
```
Comment (barzilay) [inline .gitlab-ci.yml]: Can we remove the gitlab-ci and related calliope keys in the config files? We can add the functionality to run and upload the test(results) on our gitlab environment.

File: lib/lapis_lazuli/browser.rb [hunk 2]
```diff
@@ -365,12 +341,13 @@ def create_driver(browser_wanted=nil, optional_data=nil)
             else
               world.log.debug "User-agent was already set in the :switches."
             end
-          else
-            warn "#{device} user agent cannot be set for #{b.to_s}. Only Chrome & Firefox are supported."
-        end
+         # else
```
Comment (barzilay) [inline lib/lapis_lazuli/browser.rb]: lets remove all the commented out code, or do we still need this at some point?

File: lib/lapis_lazuli/generators/cucumber/template/config/cucumber.yml [hunk 3]
```diff
@@ -1,11 +1,10 @@
 ################################################################################
-# Copyright <%= config[:year] %> spriteCloud B.V. All rights reserved.
-# Generated by LapisLazuli, version <%= config[:lapis_lazuli][:version] %>
-# Author: "<%= config[:user] %>" <<%= config[:email] %>>
+# Copyright 2017 spriteCloud B.V. All rights reserved.
```
Comment (barzilay) [inline lib/lapis_lazuli/generators/cucumber/template/config/cucumber.yml]: Should be 2018, but Wasnt  <%= config[:year] %> a good setup?

File: lib/lapis_lazuli/generators/cucumber/template/config/cucumber.yml [hunk 4]
```diff
@@ -1,11 +1,10 @@
...
+# Generated by LapisLazuli, version 2.0.0
```
Comment (barzilay) [inline lib/lapis_lazuli/generators/cucumber/template/config/cucumber.yml]: Should be <%= config[:lapis_lazuli][:version] %> right?

File: test/config/cucumber.yml [hunk 5]
```diff
@@ -24,13 +24,11 @@ chrome: BROWSER=chrome
...
 
-u: TEST_ENV=acceptatie -t @u,@uat
-uat: TEST_ENV=acceptatie -t @u,@uat
+remote: SELENIUM_ENV=remote -t ~@ignore_on_remote TEST_ENV=remote
 
-p: TEST_ENV=production -t @p,@prod
-production: TEST_ENV=production -t @p,@prod
+################################################################################
+# Calliope.pro settings
+calliope_key: API_KEY=NDM4NmY5Njk5N2U2ZGJmNmRhYmI1ZDU1YjRkNDM1NzMxYjhlMDk2MjNlOGE5MWUyOTg2ZWUyMDJiZTVlMjk2YmJh
```
Comment (barzilay) [inline test/config/cucumber.yml]: Lets add the calliope key only in gitlab.

File: .gitlab-ci.yml (hunk 1 above)
Comment (sjieg) [reply .gitlab-ci.yml]: Indeed, removed gitlab-ci file

File: lib/lapis_lazuli/browser.rb (hunk 2 above)
Comment (sjieg) [reply lib/lapis_lazuli/browser.rb]: Fixed.

File: lib/lapis_lazuli/generators/cucumber/template/config/cucumber.yml (hunk 3 above)
Comment (sjieg) [reply lib/lapis_lazuli/generators/cucumber/template/config/cucumber.yml]: Accidentally overwritten this. I've put the code back.

File: test/config/cucumber.yml (hunk 5 above)
Comment (sjieg) [reply test/config/cucumber.yml]: Removed key

**Comment by barzilay** [comment]
Good to go, lets release tomorrow!

**Pull request merged**